### ISSUE TITLE
quad.lua kills timer-1 on edgetx 2.8

### ISSUE THREAD

**Issue by r0b985** [alexey-gamov/opentx-quad-telemetry#4]
Since edgetx 2.8, quad.lua disables timer-1 on my x9 lite S radio. The timer can't be turned on while the lua is running and it disappears from the main screen (which sould be independant from the lua). To make the timer visible again I need to remove quad.lua from the telemetry screens in the display tab. Then it appears on the main screen again. So the lua seems to mess with edgetx 2.8 somehow.

**Comment by alexey-gamov** [comment]
Hi! You can change timer ID in the end of settings section: `telemetry = {satcount = 'Sats', timer = 1}`.

**Comment by r0b985** [comment]
Yes when I set up timer2 and change it accordingly in the lua, the issue remains the same. The timer on the main screen instead starts working again since it's using timer1 and doesn't mess with the lua anymore.

**Comment by alexey-gamov** [comment]
Hmm. When you set up timer2 in lua script what problem is remaining?

I will be thankful if you shoot some video with commentary and sent to my telegram account @alexey_gamov!

Originally implementation is:
- start fly timer in script if it's not defined in model
- if it exist in the model and set up for 0:00 = lua script only shows it's value with some animation
